### PR TITLE
Add install/uninstall examples yarn and npm

### PR DESCRIPTION
**Pull request by rodrigooler**

### CODE REVIEW

**Comment by sibelius** [comment]
uninstall does not make sense

File: README.md [hunk 1]
```diff
@@ -11,7 +11,24 @@ React Native Fontawesome Icons
...
+
+## Using yarn
+
+### Install
+`npm i -g yarn`
+
+`yarn add react-native-fontawesome`
+
+### Uninstall
+`yarn react-native-fontawesome`
```
Comment (sibelius) [inline README.md]: wrong

Comment (rodrigooler) [inline README.md]: Resolved.

**Comment by sibelius** [comment]
great

**Pull request merged**